test/component/testDocumentsPage.js

```diff
@@ -1,7 +1,9 @@
+// eslint-disable-line max-lines
 const TestWrapper = require('test/util/TestWrapper');
 const config = require('app/config');
 const ThankYou = require('app/steps/ui/thankyou/index.js');
 const ihtContent = require('app/resources/en/translation/iht/method');
+const {assert} = require('chai');
```

Comment:
`assert` is not being used in the code and is causing a linting error, please remove it.

Do you have the git hooks set up? Details are in the [README file](
https://github.com/hmcts/probate-frontend/blob/master/README.md)